build.gradle

```diff
@@ -28,6 +28,7 @@ repositories {
 subprojects {
     repositories {
         mavenCentral()
+        jcenter()
```

Comment:
Please, use the https://search.maven.org/artifact/org.jetbrains.lets-plot/lets-plot-batik to load the artifact, jcenter is not used anymore by many open-source projects https://jfrog.com/blog/into-the-sunset-bintray-jcenter-gocenter-and-chartcenter/

Reply:
have to change it to 
`
maven {
            url "https://dl.bintray.com/kotlin/kotlinx.html"
        }
`
because of [this issue](https://github.com/Kotlin/kotlinx.html/issues/151)